app/models/user.rb

```diff
@@ -6,4 +6,7 @@ class User < ApplicationRecord
   has_many :commented_posts, through: :comments, source: :post
 
   scope :three_recent_posts, ->(user) { user.posts.order(created_at: :desc).limit(3) }
+
```

Comment:
- [ ] It would be very nice if you test this method since it is a requirement that all the custome methods in models should be tested 👍

Reply:
@HAMMAS-SALEEM The suggested test gave me a little bit of tough time but I have learned a lot about rails app testing while doing that. Thank you Hammas your feedback was very progressive for me. 👍